databricks/sdk/core.py

```diff
@@ -981,11 +981,12 @@ def do(self,
            method: str,
            path: str,
            query: dict = None,
+           headers: dict = None,
            body: dict = None,
            raw: bool = False,
            files=None,
            data=None) -> dict:
-        headers = {'Accept': 'application/json', 'User-Agent': self._user_agent_base}
+        headers |= {'User-Agent': self._user_agent_base}
```

Comment:
a) may not work in python 3.7
b) this will crash, as headers default value is empty